src/main/java/seedu/bankwithus/Ui.java

```diff
@@ -74,12 +74,15 @@ public void printLine() {
     }
 
     public void viewAccount(String accDetails) {
-        String name = accDetails.split(";")[0];
-        String bal = accDetails.split(";")[1];
-        printLine();
-        System.out.println("Name: " + name);
-        System.out.println("Balance: $" + bal);
-        printLine();
+        String[] accounts = accDetails.split("\\n");
+        System.out.println("MAIN Account:");
+        for (String account : accounts) {
+            String name = account.split(";")[0];
+            String bal = account.split(";")[1];
+            System.out.println("Name: " + name);
+            System.out.println("Balance: $" + bal);
+            printLine();
+        }
```

Comment:
For line 77 to 85, create a new function and comand view all account for it. We want to still retain the previous function

Reply:
Not quite sure about this, you can try to resolve it later